src/js/visual/configurator-csr.js

```diff
@@ -1160,6 +1167,7 @@ ripe.ConfiguratorCsr.prototype._initCsr = function() {
 ripe.ConfiguratorCsr.prototype._deinitCsr = function() {
     this._destroyDebug();
     this._destroyInitialsResources();
+    this._destroyConfig();

```

Comment:
Hmm, this shouldn't be here, it's not CSR related, as in, it's not related to the scene building and Three.js resources stuff.

Also this is what I'm talking about, this shouldn't affect anything as the `_deinitCsr` is only called on the `deinit()` and that will clear all `owner` binds

Reply:
> Also this is what I'm talking about, this shouldn't affect anything as the _deinitCsr is only called on the deinit() and that will clear all owner binds

Can you explain how the `deinit()` clears all owner binds?

I don't see that happening and I couldn't find the logic of where this happens.

Here is a video of flawless model swap in RIPE White using CSR:

https://user-images.githubusercontent.com/24736423/222774339-f6ff6a91-f451-4d75-9357-5a71c3dc7b8b.mp4

Here is another video with the same setup the only difference being that I removed the line of code that calls `this._destroyConfig();`:

https://user-images.githubusercontent.com/24736423/222774883-1b219828-4158-48b0-a1b6-e22e1935c356.mp4

This is to show you that these binds:
```

    this._onPreConfigBind = this.owner.bind("pre_config", (brand, model, options) => this._onPreConfig(this));
    this._onPostConfigBind = this.owner.bind("post_config", config => this._onPostConfig(this, config));
```

need to be properly unbinded, specially after that configurator stops to exist for some reason and at this current implementation they aren't.

What's your suggestion on fixing this considering that now we know that we need to do these unbindings when doing `_deinitCsr`?

Reply:
Here is a simple example:

The button just calls configurator `.deinit()` and you can see that the callbacks are all cleared because `callbacks` is set to `null` in the `Observable` `deinit()` call


https://user-images.githubusercontent.com/22588915/223089000-651ea97c-ca82-46a9-9892-fa2897b99e4a.mp4

Change file format to `.html`
[csr.txt](https://github.com/ripe-tech/ripe-sdk/files/10897257/csr.txt)